notebooks/triposr-3d-reconstruction/triposr-3d-reconstruction.ipynb

```diff
@@ -0,0 +1,762 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        'head_mask': torch.tensor([0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0])
</pre><p>is it possible to remove head_mask? as it is not used in model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>removed</p>